test/test_transforms.py

```diff
@@ -2018,6 +2033,8 @@ def input_img(self):
         return input_img
 
     def test_affine_translate_seq(self, input_img):
+        # TODO: LOL, wait, we support np arrays???? (see input_img fixture above)
+        input_img = torch.randint(0, 256, size=(224, 224), dtype=torch.uint8)
```

Comment:
We don't. `numpy` is used to generate the starting point and expected result. Our transform only gets passed the `pil_image` fixture, which is what it says it is. v2 tests are passing here. Was this just a WTF comment or is there actually something to do here?